library/Cube/DbCube.php

```diff
@@ -130,11 +130,10 @@ public function getDbName()
      *
      * @return \Zend_Db_Select
      */
-    public function innerQuery()
+    public function getInnerQuery()
```

Comment:
Please be careful when changing public methods in the Cube. The Cube provides Hooks for custom Actions (see `library/Cube/Hook/ActionsHook.php' ). Those Hooks receive the Cube instance, and public methods on the Cube make part of the API they might be expecting. Blind guess: this change breaks the Cube Action Hook implementation of the Icinga Director

Reply:
Oh, I wasn't aware of that.
Thanks for pointing it out!

I will revert the name refactor :)